mantidproject/mantid
# Cleaning up the sphinx documentation as discussed at dev meeting

I want to close issue #21786. This branch is the beginning of work to clean up and clarify the documentation. The Sphinx documentation has been updated to:

* Include a section on training and tutorials
* Ensure that the documentation link at the top of each page returns to the top of the Sphinx docs page
* Add date stamps to pages
* Remove low resolution image from the front page

**To test:**

Build new documentation and make sure that:

* All top level links work
* Directory structure makes sense

**Release Notes** 

This does not need to be included in the release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
